src/accelerate/utils/modeling.py

```diff
@@ -1231,6 +1231,54 @@ def load_state_dict(checkpoint_file, device_map=None):
         return torch.load(checkpoint_file, map_location=torch.device("cpu"))
 
 
+def get_state_dict_offloaded_model(model: nn.Module):
+    """
+    Returns the state dictionary for an offloaded model via iterative onloading
+
+    Args:
+        model (`torch.nn.Module`):
+            The offloaded model we want to save
+    """
+    state_dict = {}
+    placeholders = []
+    for name, module in model.named_modules():
+        if name == "":
+            continue
+        if (
+            hasattr(module, "_hf_hook")
+            and str(module._hf_hook).startswith("AlignDevicesHook")
```

Comment:
Let's try to fix the import issue =) 
```suggestion
            and isinstance(module._hf_hook,AlignDevicesHook)
```